CHANGELOG.latest.md

```diff
@@ -1 +1,2 @@
-- Update purchases-android to 5.3.0 (https://github.com/RevenueCat/purchases-android/releases/tag/5.3.0)
+- Fixed snake_case and camelCase serialization inconsistency in `StoreProduct` (https://github.com/RevenueCat/purchases-hybrid-common/pull/187)
```

Comment:
could we get an explicit list of updated fields so it's easier for us when upgrading? That way we don't have to dig through the PR to figure out which types have been changed

Reply:
Good point.